builder/main.go

```diff
@@ -23,7 +23,7 @@ func main() {
 
 	runner := buildpackrunner.New(&config)
 
-	err := runner.Run()
+	_, err := runner.Run()
```

Comment:
My understanding is that the buildpacks team will take ownership of the lifecycle moving forward, and that future pull requests will be reviewed by us.

The worst thing that happens if someone removes the extra return value is that we would need to add it back before updating the copy of the lifecycle vendored by the multi-buildpack.

I agree that unit tests are desirable here. I suspect we will add them as we port the BAL to Windows.